docs/reference/mouse_bindings.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Mouse bindings
+---
+
+The mousemap files (which have the extension `.sublime-mousemap`)
+control what commands are executed when a user performs an action
+with a mouse e.g. clicking a mouse button, scrolling the scroll
+wheel etc.
+
+
+## File Format
+
+Here is a small excerpt from `Default/Default (Windows).sublime-mousemap`.
+
+```json
+[
+	// Basic drag select
+	{
+		"button": "button1", "count": 1,
+		"press_command": "drag_select"
+	},
+	{
+		"button": "button1", "count": 1, "modifiers": ["ctrl"],
+		"press_command": "drag_select",
+		"press_args": {"additive": true}
+	},
+	{
+		"button": "button1", "count": 1, "modifiers": ["alt"],
+		"press_command": "drag_select",
+		"press_args": {"subtractive": true}
+	},
+]
+```
+
+The following are the keys that a single JSON entry 
+in a mousemap file can take.
+
+`button`
+: The name of the button.
+
+  This defines the name of the button.
+  E.g. `button1` defines the left mouse button, `button2`
+  defines the right mouse button etc.
+  Similarly, if a command has to be executed when the scroll
+  wheel scrolls, then we have `scroll_up` & `scroll_down`.
+
+`modifiers`
+: A list of modifier keys.
+
+  This defines a list of modifiers keys that have to 
+  be pressed simultaneously (along with the corresponding button click)
+  for the command to be triggered. E.g. `["ctrl", "alt"]`.
+
+`command`
+: The `command` to execute.
+
+  This defines the command to be executed when the corresponding
+  button is **released after being pressed**.
+
+`args`
+: The arguments for `command`.
+
+  This is a dictionary of arguments to be passed
+  on to the `command`.
+
+`press_command`
+: The `press_command` to execute.
+
+  This defines the command to be executed when the corresponding
+  button is **pressed**. Note that this is different from `command`,
+  in the sense that it is triggered when a user presses the corresponding 
+  mouse button, while the former is executed when the user releases the 
+  said button after it is pressed.
+
+`press_args`
+: The arguments for `press_command`.
+
+  This is a dictionary of arguments to be passed
+  on to the `press_command`.
+
+`count`
```

Comment:
The list of keys seems to be unsorted. I'd expect it to be sorted either conceptually (triggers first, then results) or alphabetically.

Reply:
Can you explain a bit more ? Not really understood what do you mean ?

Reply:
I mean that there doesn't seem to be neither a group nor a consistent ordering for the keys explained here.

Reply:
As far as I know, there is no particular order in which the keys should appear. At least, in key bindings, you can reorder them in any way you want and they'd still work.